### ISSUE TITLE
Test User Flow and Wireframes with Initial Customers

### ISSUE THREAD

**Issue by michaelsena** [uport-project/3box#76]
* Bounties
* TMNT

**Comment by dazuck** [comment]
stretch goal

**Comment by michaelsena** [comment]
I imagine this is where the voiceover demo will go. We can probably get direct feedback from Mark and Bryan using that, right? Will upload tomorrow.

**Comment by michaelsena** [comment]
Jarad gave it a :+1:.

Still waiting on Beylin and Bryan.

**Comment by michaelsena** [comment]
Sent an email to everyone we interviewed with an update. Closing this story now.
